static/js/lib/ckeditor/plugins/MarkdownSyntaxPlugin.ts

```diff
@@ -29,12 +29,12 @@ export default abstract class MarkdownSyntaxPlugin extends MarkdownConfigPlugin
         ...currentConfig.showdownExtensions,
         this.showdownExtension
       ],
-      turndownRules: [...currentConfig.turndownRules, this.turndownRule]
+      turndownRules: [...currentConfig.turndownRules, ...this.turndownRules]
     }
     this.setMarkdownConfig(newConfig)
   }
 
   abstract get showdownExtension(): () => Showdown.ShowdownExtension[]
 
-  abstract get turndownRule(): TurndownRule
+  abstract get turndownRules(): TurndownRule[]
```

Comment:
maybe it's getting too fiddly, but I tested this out locally and it would work for allowing _no boilerplate at all_ on subclasses of `MarkdownSyntaxPlugin`

```suggestion
  get turndownRules(): TurndownRule[] {
    return []
  }
```

then basically subclasses that want to set a turndown rule would override this method, and others wouldn't. We could make the same change to 'de-abstract' the showdown method if we wanted as well

anyway - consider that a fully optional suggestion 😄

Reply:
I'm going to leave as-is for now because I like the idea that you have to opt-in to "no, I really don't need Turndown rules for this extension". Maybe that's silly—we would probably catch that sort of error in the unit tests 🤷